syntax = "proto3";

message RequestMessage {
	int32 id = 1;
	string interface = 2;
	int32 money = 3;
}


message ResponseMessage {
	string interface = 1;
	string result = 2;
	int32 money = 3;
}

service Branch {
	rpc MsgDelivery(RequestMessage) returns (ResponseMessage) {}
	rpc MsgPropagate(RequestMessage) returns (ResponseMessage) {}
}